### ISSUE TITLE
what is the most stable branch to deploy?

### ISSUE THREAD

**Issue by lstan44** [pH7Software/pH7-Social-Dating-CMS#1150]
hey guys, i'm attempting to run this locally to try it out and I'm wondering which branch is the most stable.
i cloned the `18.x` branch and tried to build the docker image and it failed.

**Comment by El3ctra** [comment]
I also had issues deploying with deploying 18.X. And the 17.X worked like a charm.
You should give it a try.

**Comment by pH-7** [comment]
Yes, that’s correct. v17 was more stable. v18 hasn’t reached that same level yet, mainly because there wasn’t enough time for full testing before release.

With v17, I had much more time to test and make sure there were no bugs. That wasn’t fully possible with v18.

I’m currently working on v18 to improve stability and make it more robust, with the goal of reaching the same level of reliability as v17.
